### DISCUSSION TITLE
Is there any way I can turn off chakra ui console log?

### DISCUSSION THREAD

**Discussion by quangtho2910** [chakra-ui/chakra-ui#6740] [Q&A] [answered]
Hello everyone,

I just tried Chakra UI for a couple days with Next js. And I would like to ask if I can turn off the Chakra UI console log in next js dev mode? There are thousands of messages in the first-page load!!!
Any help or advice is appreciated!

![スクリーンショット 2022-09-22 164538](https://user-images.githubusercontent.com/3017528/191688245-9514ba8e-ca75-4b09-af7b-2be3e48c3872.png)

**Comment by khufu-adecoster** [comment, marked as answer]
I was having the same problem, I updated the versions of my chakra ui packages and it's not displaying it anymore 👌

**Reply by quangtho2910** [reply]
Thank you so much. Let me update my package!
